### DISCUSSION TITLE
How "non coders" can help?

### DISCUSSION THREAD

**Discussion by RicoLaaa** [mawoka-myblock/ClassQuiz#170] [Ideas]
Hi!

I just found ClassQuiz a couple of days ago, and I'm already trying to push it to teachers and get them to use it instead of Kahoot!
I was wondering, how can "non-coders" help with the app development? Would like an interface mokup on figma help in any way?
Where is the best place to ask for features? Here in the discussion or in "issues" (like ability to format text (bold/italic/etc), ability to add videos and not only images, have bigger buzzer buttons like in Kahoot!, tooltips display when I hover a button, etc...

The app is already working great, although it looks less "sexy" than its closed source counterparts

Thanks for this excellent project!

**Comment by mawoka-myblock** [comment]
First, thank you! These smaller things (like tooltips, new button-styles, etc) can go into separate issues. Improving the overall design isn't necessary in my opinion, but I am open to suggestions (either mockup or describing). Finding bugs and telling me what is missing for you is help enough! You can also translate ClassQuiz if you want.

**Reply by RicoLaaa** [reply]
Thanks for your reply. Il post the different issues and make some mokups. Already finished translating french las night, and started the spanish one ;)

**Comment by RicoLaaa** [comment]
@mawoka-myblock spanish translation done 🇪🇸
